.travis.yml

```diff
@@ -8,7 +8,7 @@ services:
 
 branches:
   only:
-    - master
+#    - master
```

Comment:
Note, I'm fine with just turning off Travis, I think it's been running in parallel long enough.

Reply:
When merging, I'll disable it within Travis (also need to cherry-pick it into v2.3 release branch then).